src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -528,6 +528,11 @@ public String getPagedListResponseFactoryName(Method method) {
     return privateFieldName(Name.upperCamel(method.getSimpleName(), "PagedListResponseFactory"));
   }
 
+  /** The variable name of the gRPC request object. */
+  public String getRequestObjectName(Method method) {
+    return getNotImplementedString("SurfaceNamer.getRequestObjectName");
+  }
```

Comment:
Maybe instead name this `getRequestVariableName` for consistency with other methods implementing the same kind of thing. Also, it seems like you could provide a reasonable default implementation user using `localVarName`, similarly to `getApiWrapperVariableName`.